WebHostLib/static/assets/player-settings.js

```diff
@@ -145,8 +145,13 @@ const buildOptionsTable = (settings, romOpts = false) => {
         range.setAttribute('min', settings[setting].min);
         range.setAttribute('max', settings[setting].max);
         range.value = currentSettings[gameName][setting];
+        const settingMax = settings[setting].max;
+        const specialMax = settings[setting].specialMax;
```

Comment:
You must check for the existence of this special property, as it is not guaranteed to be present on normal range options.

Reply:
That check is done where it is referenced. Should I also check it here if it's checked later?

Reply:
If the property doesn't exist, attempting to access it will result in an an Error being thrown. You should ensure it exists before attempting to access it for the first time, which happens during assignment here. When using it later, you won't need to check for its existence again.